capdec.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] Kindly check on the functionality the  correct_name method is supposed to capitalize the first letter and not the entire name.

Reply:
thank you